src/NosCore.GameObject/Providers/ItemProvider/Item/WearableInstance.cs

```diff
@@ -70,8 +70,7 @@ public void SetRarityPoint()
                 case EquipmentType.MainWeapon:
                 case EquipmentType.SecondaryWeapon:
                 {
-                    var point = CharacterHelper.Instance.RarityPoint(Rare,
-                        Item.IsHeroic ? (short) (95 + Item.LevelMinimum) : Item.LevelMinimum);
+                    var point = 0; //todo CharacterHelper.Instance.RarityPoint(Rare, Item.IsHeroic ? (short) (95 + Item.LevelMinimum) : Item.LevelMinimum);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5574106)